File: fitness-backend/food-microservice/src/main/java/org/mwdziak/dto/FoodDTO.java
package org.mwdziak.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.RequiredArgsConstructor;

import java.util.List;

@Data
@RequiredArgsConstructor
@AllArgsConstructor
@Builder
public class FoodDTO {
    private Long fdcId;
    private String description;
    private String foodKind;
    private FoodNutrientDTO nutrients;
}

File: fitnessmobileclient/app/src/main/java/com/mwdziak/fitness_mobile_client/dto/FoodDTO.kt
package com.mwdziak.fitness_mobile_client.dto

import kotlinx.serialization.Serializable

@Serializable
data class FoodDTO(
    val fdcId: Int,
    val foodKind: String,
    val description: String,
    val calories: Double,
    val protein: Double,
    val carbohydrates: Double,
    val fat: Double
)